Makefile.cross-compiles

```diff
@@ -2,7 +2,7 @@ export PATH := $(GOPATH)/bin:$(PATH)
 export GO111MODULE=on
 LDFLAGS := -s -w
 
-os-archs=darwin:amd64 darwin:arm64 freebsd:386 freebsd:amd64 linux:386 linux:amd64 linux:arm linux:arm64 windows:386 windows:amd64 linux:mips64 linux:mips64le linux:mips:softfloat linux:mipsle:softfloat
```

Comment:
Not inclined to add `openbsd:386`.

You can remove `freebsd:386` by the way.

Reply:
why not build for x86 bsd? i still have some 32-bit machines.

Reply:
You can build it by yourself. But we are tend to encourage upgrading to 64-bit architectures in release assets.